apps/fishing-map/features/app/App.tsx

```diff
@@ -197,7 +197,7 @@ function App() {
       if (reportAreaBounds) {
         fitMapBounds(reportAreaBounds, { padding: FIT_BOUNDS_REPORT_PADDING })
       } else {
-        setMapCoordinates({ latitude: 0, longitude: 0, zoom: 0 })
+        setViewState({ ...viewState, latitude: 0, longitude: 0, zoom: 0 })
```

Comment:
I'd use the [useSetAtom](https://jotai.org/docs/core/use-atom#usesetatom) to avoid re-rendering the entire app on every viewState change:
```
This is especially useful when the performance is a concern, as the const [, setValue] = useAtom(valueAtom) will cause unnecessary rerenders on each valueAtom update.
```
See [this commit](https://github.com/GlobalFishingWatch/frontend/commit/24fda5c4eac0ae0d99845520902a08549a1ebc68) as an example of how to avoid it